contentcuration/contentcuration/frontend/channelEdit/views/CurrentTopicView.vue

```diff
@@ -538,44 +538,47 @@
         }
 
         // All sources should be from the same region
-        const sourceRegion = drop.sources[0].region;
+        const sources = drop.sources || [];
+        const sourceRegion = sources.length > 0 ? sources[0].region : null;
         const payload = {
           target: identity.metadata.id,
           position,
         };
 
-        // When the source region is the clipboard, we want to make sure we use
-        // `excluded_descendants` by accessing the copy trees through the clipboard node ID
-        if (sourceRegion && sourceRegion.id === DraggableRegions.CLIPBOARD) {
-          return Promise.all(
-            data.sources.map(source => {
-              // Using `getCopyTrees` we can access the `excluded_descendants` for the node, such
-              // that we make sure to skip copying nodes that aren't intended to be copied
-              const trees = this.getCopyTrees(source.metadata.clipboardNodeId, true);
+        if (payload.target) {
```

Comment:
In such scenarios where you have to nest a large block of code in a conditional, it can make more sense to have to shortcut the code early instead, which keeps the code flatter. For example:
```js
if (!payload.target) return;
```